city_name.py

```diff
@@ -0,0 +1,45 @@
+from flask import Flask, request, Response, jsonify, abort
+from fuzzywuzzy import process
+
+with open("/home/meghanabhange/mysite/prefered_labels.txt", "r") as f:
```

Comment:
How can we not deploy hard coded paths on `PythonAnywhere` - so that we can change the provider if we have to? 
Maybe use relative paths with `Pathlib` so that we can deploy cleanly?

Reply:
Resolved by 684e2d5